panels/display/cc-display-panel.c

```diff
@@ -2298,20 +2300,23 @@ show_setup_dialog (CcDisplayPanel *panel)
     }
 
   /* overscan */
-  priv->scaling_check = gtk_check_button_new_with_label (_("Adjust screen for TV"));
-  gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (priv->scaling_check),
-                                gnome_rr_output_info_get_underscanning (priv->current_output));
-  g_signal_connect (G_OBJECT (priv->scaling_check), "toggled",
-                    G_CALLBACK (checkbutton_toggled), panel);
+  if (!g_str_has_prefix (gnome_rr_output_info_get_name (priv->current_output), INTERNAL_DISPLAY))
```

Comment:
@cosimoc I'll do!

That function is not listed in the documentation I was checking. Probably it was outdated.